src/Apps/CollectorProfile/Routes/Saves2/Components/ArtworkListEmptyState.tsx

```diff
@@ -18,24 +18,26 @@ export const ArtworkListEmptyState: FC<ArtworkListEmptyStateProps> = ({
   return (
     <Flex
       flex={1}
+      flexDirection={["column", "row"]}
       justifyContent="space-between"
       alignItems="center"
       bg="black5"
       p={2}
     >
       <Box>
-        <Text variant="sm-display">{text.title}</Text>
-        <Text variant="sm-display" color="black60">
+        <Text variant={["sm", "sm-display"]}>{text.title}</Text>
+        <Text variant={["sm", "sm-display"]} color="black60">
           {text.description}
         </Text>
       </Box>
 
-      <Spacer x={2} />
+      <Spacer x={[0, 2]} y={[2, 0]} />
 
       <Button
         // @ts-ignore
         as={RouterLink}
-        variant="secondaryBlack"
+        width={["100%", "auto"]}
+        variant="primaryBlack"
```

Comment:
Just wanted to confirm that this is the correct variant, compared to the one in [the Figma screen cap on the card](https://www.notion.so/artsy/UX-mWeb-Empty-state-is-different-from-Figma-design-c1dcb05813fc45d9aec709b57c2fa68c)? 

(Screen cap looks like `primaryGray`? But if not, feel free to merge.)

Reply:
> Just wanted to confirm that this is the correct variant, compared to the one in [the Figma screen cap on the card](https://www.notion.so/artsy/UX-mWeb-Empty-state-is-different-from-Figma-design-c1dcb05813fc45d9aec709b57c2fa68c)?

yep, it's correct button variant (we discussed it today with Ryan)

Reply:
> I've just realised the primary grey button has been removed from the design system

Received message from Ryan today